lib/rules/function-url-no-scheme-relative/index.js

```diff
@@ -25,7 +25,7 @@ const rule = function(actual) {
 			functionArgumentsSearch(decl.toString().toLowerCase(), 'url', (args, index) => {
 				const url = _.trim(args, ' \'"');
 
-				if (!isStandardSyntaxUrl(url) || url.indexOf('//') !== 0) {
+				if (!isStandardSyntaxUrl(url) || !url.startsWith('//')) {
```

Comment:
This needs double checking but I assume this was the intention

Reply:
If there's no test coverage it would be great to add some to be sure

Reply:
It is covered already, but mistakes happen :)